test/dotnet-new.Tests/Approvals/DotnetNewDetailsTest.CanDisplayDetails_InstalledPackage_OtherFeed.verified.txt

```diff
@@ -1,41 +1,15 @@
-﻿Microsoft.Azure.WebJobs.ItemTemplates
-   Package version: 4.0.3038
-   Details: Collection of templates for Azure Functions
+﻿NUnit3.DotNetNew.Template
+   Package version: 1.9.0
```

Comment:
Is there any way to change the test comparison logic to not actually check the details and package version? If we don't control the template, those could change at any time and break the test (even if we do control it, it might need to change). Having a test gate on a dynamic set of values like that is not great.

Reply:
I noticed that almost all the test failures are due to version issues, so we can simply check if it is the latest version.